src/swagger2har.js

```diff
@@ -142,6 +146,12 @@ var getResolvedSchema = function(swagger, schema) {
  */
 var getBaseUrl = function(swagger) {
   var baseUrl = ""
+
+  if (isOAS) {
+    baseUrl = swagger.servers[0].url
+    return baseUrl
+  } 
```

Comment:
This might be problematic when there is more than one server url

Reply:
why not

```
return swagger.servers[0].url
``` ?

Reply:
Logged in Jira.